domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3119,6 +3112,13 @@ private void execute(String[] sql, boolean stopOnError,
     });
   }
 
+  static void startPostgresProcess(EmbeddedPostgres e) {
+    Optional<PostgresProcess> p = e.getProcess();
+    if (p.isPresent()) {
+      postgresProcess = p.get();
+    }
+  }
```

Comment:
It looks like this method does not start the PostgreSQL process, rather it remembers the process that has just been started (in order to allow for repeated execution of `startEmbeddedPostgres` with only one embedded PostgreSQL started).

Is that a sensible interpretation?

If so, could this instead be named something like `rememberEmbeddedPostgresProcess`? 

(it can also be made private)